terraform/variables.tf

```diff
@@ -23,7 +23,7 @@ variable "max_size" {
 # Route 53 related variables
 variable "domain" {
   description = "Domain name hosted by Route 53"
-  default     = "modus.app"
+  default     = "moduscreate.com"
```

Comment:
The `modus.app` domain is currently pointed at CloudFlare so we had to switch this.